mappings/net/minecraft/server/MinecraftServer.mapping

```diff
@@ -1,4 +1,20 @@
-CLASS net/minecraft/server/MinecraftServer
+CLASS net/minecraft/server/MinecraftServer net/minecraft/server/MinecraftServer
+	COMMENT Represents a logical Minecraft server.
+	COMMENT
+	COMMENT <p>Since Minecraft uses a Client-Server architecture for the game, the server processes all logical game functions.
+	COMMENT A few of the actions a Minecraft server will handle includes processing player actions, handling damage to entities, advancing the world time and executing commands.
+	COMMENT
+	COMMENT <p>There are two primary implementations for a Minecraft server: a dedicated and an integrated server.
+	COMMENT
+	COMMENT <p>A dedicated server is a Minecraft server not attached to a Minecraft game client and may be run remotely from any connected players.
+	COMMENT A dedicated server has a few exclusive features such as a whitelist/blacklist, remote rcon connections, and a terminal to input commands.
+	COMMENT
+	COMMENT <p>An integrated server is functionally equivilant to a dedicated server except that is a hosted by a Minecraft game client and is typically used in a Singleplayer world.
+	COMMENT An integrated server differs from a dedicated server by allowing connections within the local area network (LAN) and the host client.
+	COMMENT Generally you will always want to treat connection to a integrated server like you would to a dedicated server since the concept of an integrated server is an implementation detail in Minecraft.
```

Comment:
```suggestion
	COMMENT Generally, you will always want to treat connection to an integrated server like you would to a dedicated server since the concept of an integrated server is an implementation detail in Minecraft.
```